enterprise/lms_api.py

```diff
@@ -24,24 +26,36 @@
     CourseEnrollment = None
 
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
@pomegranited here's the thing - on one hand, pylint demands capitalizing this as it thinks it is a constant; on the other hand it's actually not a constant, but an object. 

I believe other files in this repo [suppressed the error in this case](https://github.com/edx/edx-enterprise/blob/4d78623a3897d2eb8614bb30fc994ccda9d26ae0/enterprise/signals.py#L12), so would be quite nice if you could do the same.